sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1128,16 +1128,16 @@ private static string CreateContinuationTokenFromHeaders(TableQueryEntitiesHeade
             }
         }
 
-        private static (string NextPartitionKey, string NextRowKey) ParseContinuationToken(string continuationToken)
+        internal static (string NextPartitionKey, string NextRowKey) ParseContinuationToken(string continuationToken)
         {
             // There were no headers passed and the continuation token contains just the space delimiter
-            if (continuationToken?.Length <= 1)
+            if (continuationToken is null || continuationToken.Length <= 1)
```

Comment:
nit: I think this change is equivalent to what was there before ([example](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRjz1QMwAIA3AQwBsBLOAZWDHqgHNyBecgImABTAM7A+AbhJlyUCLVos2nHjLm1JuPPQBmAChoNmrdhwD8AOgAygzsAAW5AHzlMASjwBvPOR/lSATl0+A0ZFE3J6YRkAe2BVeT5XDQBfYlwdXVl5MM4La1sHZzdPb18AoKzaP2lImLjKxJS8QVphQRLcXx9yvkqcrlqGpN88VNwgA=))

Reply:
Perhaps I am misunderstanding but the case I was protecting against was the `null` continuation token causing a NRE on the `Split`.
Before: [SharpLab](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYCMBYAUH1TATgAoAFAQzAGcBTAYQHspgBLKCCt5gFUYGtaUEhwA2ogJQSA3HgKYAbAAIShAAxKAcrQAewSmDbcoAaVoBPGEvVbdwAEqMA7mfMSlBukxbtOxvoLCNgDGzGwcXKy8AkISeADeeErJSgD0qUo8ABa0YLRKTrn5UIxKORRwudRKAA4U1HRwShRQTcA5SqE+EcZKwDFQnWEU7NUAVhDUwH0d1HXB+ZWirAC2rMC5SSmsAGYqXeF+UVABQgD8AHQAMkIA5u1KADwAvEqYcbgpSomfXymoAHYVGJRFYQTItskAL5yX7JABuVD6A2qrwOvki0UCFwAyjVlsASAByJREiFw6xAkj9QLUADaagAulYaUJqNc7g8AHxvJRnZG0umYRlKEBKcGyXBQoA==)
After: [SharpLab](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYCMBYAUH1TATgAoAFAQzAGcBTAYQHspgBLKCCt5gFUYGtaUEhwA2ogJQSA3HgKYAbAAIShAAxKAcrQAewSmDbcoAaVoBPGEvVbdwAEqMA7mfMSlBukxbtOxvoLCNgDGzGwcXKy8AkISeADeeErJSgD0qUo8ABa0YLRKTrn5UIxKORRwudRKAA4U1HRwShRQTcA5SqE+EcZKwDFQnWEU7NUAVhDUwH0d1HXB+ZWirAC2rMC5SSmsAGYqXeF+UVABQkqs1WKiSsjIQ91H0YEAdAAyQgDm7UoAPAC8SkwcVwKSUiRBoJSqAA7CorlYrjItskAL5yCHJABuVD6A2qAIOvkiTyEzwAyjVlsARLQnABtAC6YKUAHJWUoUVZ0EiMdZYSR+oFqHS1AyrIKhNQ3p9vgA+QFKABkitxQrpmAZ0qgXyySnlGgA/KrJeqmSAlIjZLgUUA=)

Per [StackOverflow](https://stackoverflow.com/a/15777768) and linked docs:
> For the [comparison operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/comparison-operators) `<`, `>`, `<=`, and `>=`, if one or both operands are `null`, the result is `false`; otherwise, the contained values of operands are compared. Do not assume that because a particular comparison (for example, `<=`) returns `false`, the opposite comparison (`>`) returns `true`.

Reply:
Sorry, you are 100% correct. I missed that we want a true evaluation on the null but not on the `<=`